webapp/src/app.tsx

```diff
@@ -1921,9 +1923,24 @@ function initSerial() {
     }
     ws.onmessage = (ev) => {
         try {
-            let msg = JSON.parse(ev.data) as pxsim.SimulatorMessage;
-            if (msg && msg.type == 'serial')
-                window.postMessage(msg, "*")
+            let msg = JSON.parse(ev.data) as pxsim.SimulatorSerialMessage;
+            if (msg && msg.type == 'serial') {
```

Comment:
Can we move this logic into a common helper function?

Reply:
Sure.  Where can it go, such that it will be accessible by app.tsx and pxtwinrt/serial.ts?

Reply:
pxtlib is loaded in both cases (see https://github.com/Microsoft/pxt/blob/master/pxtwinrt/deploy.ts#L3). You can have a "serial" object in there. THere might be something already.

Reply:
Great, will do.

Reply:
Done.